### PR TITLE
Remove patch process

### PR DESCRIPTION
**Pull request by sandertan**
- Remove patch process of portal.properties
- Update documentation accordingly
- Remove some MSK and TCGA specific information
- Add portal.properties.EXAMPLE of current version of cBioPortal

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,13 +32,11 @@ WORKDIR $PORTAL_HOME
...
 # add buildtime configuration
-COPY ./portal.properties.patch /root/
+COPY ./portal.properties src/main/resources/portal.properties
 
 # install default config files, build and install, placing the scripts jar back
 # in the target folder where import scripts expect it after cleanup
-RUN cp src/main/resources/portal.properties.EXAMPLE src/main/resources/portal.properties \
-	&& patch src/main/resources/portal.properties </root/portal.properties.patch \
-	&& cp src/main/resources/log4j.properties.EXAMPLE src/main/resources/log4j.properties \
+RUN cp src/main/resources/log4j.properties.EXAMPLE src/main/resources/log4j.properties \
```
Comment (pieterlukasse) [inline Dockerfile]: this makes it harder to overwrite the log4j settings with an external file. Please add `COPY` command above, similar to what was done for portal.properties.

File: docs/adjusting_portal.properties_configuration.md [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
+When building an image on which to run containers, the Dockerfile adjusts configuration by copying main cBioPortal configuration file `portal.properties` to the image. To build an image that uses different settings (see the documentation on the [main properties](https://github.com/cBioPortal/cbioportal/blob/master/docs/portal.properties-Reference.md) and the [skin properties](https://github.com/cBioPortal/cbioportal/blob/master/docs/Customizing-your-instance-of-cBioPortal.md)), you can follow the steps listed below.
 
 ### Step 1: modify `portal.properties` in the cBioPortal Docker directory
-First, the  `portal.properties.EXAMPLE` of the current version of cBioPortal listed in the `Dockerfile` must be downloaded twice in the `cbioportal-docker` folder. This can be done by following these commands:
+To modify the configuration for your own cBioPortal instance, please add modifications to `portal.properties`. This file already contains several modifications for this Docker setup.
```
Comment (pieterlukasse) [inline docs/adjusting_portal.properties_configuration.md]: and log4j.properties.

**Review by pieterlukasse — CHANGES_REQUESTED** [review]
Really nice :+1: 
There is 1 small issue I commented on regarding log4j.properties

**Review by pieterlukasse — APPROVED** [review]
great! Happy to see this configuration part simplified now!

**Comment by sandertan** [comment]
Rebased and add `Update portal.properties(.EXAMPLE) to v1.11.2`-commit

**Comment by fedde-s** [comment]
Would this make it easier to document the steps of adjusting your config to the options supported by another branch/release, somewhere in [this file](https://github.com/thehyve/cbioportal-docker/blob/3ad3f4281761e4121c0cf690d554bc09ea3886d0/docs/adjusting_Dockerfile_configuration.md#build-an-image-based-on-a-different-branch)?

**Comment by fedde-s** [comment]
It does become easier to have such mismatches go completely unnoticed, I imagine.

**Comment by fedde-s** [comment]
Nice, merging those two config docs makes them a lot easier to find.

**Comment by fedde-s** [comment]
@sandertan, would you like me to release this or are you still tweaking something?

**Comment by sandertan** [comment]
@fedde-s go ahead!

**Comment by sandertan** [comment]
@fedde-s small naming suggestion: Renaming `cbioportal-image` to `cbioportal`. I accidently changed it in documentation already, so now it's inconsistent....

**Comment by fedde-s** [comment]
I included the word ‘image’ on purpose, as I imagined it might be difficult for people who've never used Docker before to know whether they're dealing with an image or a container if they have the same name. Not sure if there's a better solution for that.

**Comment by sandertan** [comment]
Indeed, in this case, it might be best to help the new people!

**Pull request merged**